requirements.py

```diff
@@ -69,6 +69,14 @@
     'crate': [  # crate databases
         'crate'
     ],
+    'central_platform': [
```

Comment:
This should just use web rather than adding this here.

Reply:
Yes, I originally did it that way but thought that having an explicit list of dependencies for central_platform would be helpful. But this does lead to code bloat. I'll remove this and just use web in the Dockerfile.